db/migrate/20250606142909_drop_forums.rb

```diff
@@ -0,0 +1,15 @@
+class DropForums < ActiveRecord::Migration[7.0]
+  def change
+    drop_table :forums
+    drop_table :moderatorships
+    drop_table :posts
+    drop_table :topic_categories
+    drop_table :topics
+    drop_table :user_topics
+
+    safety_assured do
+      remove_columns :settings, :forum_enabled, :forum_public, :anonymous_posts_enabled
+      remove_columns :users, :posts_count
+    end
```

Comment:
I think it's better to not remove columns in this PR. Removing columns it's always a problem. I'm not sure if affected tables have to be re-created, but I suspect it will cause problems on deploy.

Strong Migrations suggests to ignore columns on the model. I think that's better. Just ignore columns for now and in the future we can remove the columns in another PR if we want.